src/main/kotlin/widgets/svgButton/view/SvgButton.kt

```diff
@@ -0,0 +1,82 @@
+package widgets.svgButton.view
+
+import afester.javafx.svg.SvgLoader
+import tornadofx.*
+import javafx.scene.Group
+import javafx.scene.layout.StackPane
+import javafx.scene.paint.Paint
+
+
+//Class to create circle button with SVG graphic inside
+//Use "::functionName" to pass function as parameter
+//Example how to use: SvgButton(64.0, c("#CC4141"), "forwardArrowIcon", operation = ::abc)
+//For suggestions the best values to put in the parameter for certain buttons, scroll to bottom of this file
+class SvgButton(var buttonSize: Double= 64.0,
+                var myVariable: Paint = c("#ffffff"),
+                var svgFileName: String,
+                var svgScaleXCompareToButton: Double = 0.50484,
+                var svgScaleYCompareToButton: Double = 0.50484,
+                operation: () -> Unit ): StackPane() {
```

Comment:
can you explain your reasoning behind the operation parameter?

Reply:
the Operation parameter is necessary to make the button universally used. because SvgButton can be used in multiple instances, the operation inside the "action{  }" can't be hard-coded.